assets/base.css

```diff
@@ -1039,11 +1039,20 @@ summary::-webkit-details-marker {
   width: calc(50% - var(--grid-mobile-horizontal-spacing) / 2);
 }
 
-@media screen and (min-width: 750px) {
+@media screen and (max-width: 990px) {
+  .grid--1-col-tablet .grid__item {
```

Comment:
@kmeleta Since `grid---col-tablet` is being used in other spots like the footer, collapsible content etc I think I should just create a new naming (we can use your suggestion `.grid--#-col-tablet-down`) to account for this case in tablet and mobile and in the future we can do a refactor. Otherwise we will have to test this in many sections across the theme and ensure nothing else is broken.

I will revert the media query and give a new class for these sections using the grid section.

Reply:
Well I guess that's basically my point, I'm not sure we should change the way the grid classes/breakpoints work for _some_ cases. Not to say one way is wrong and one is right, just that we _should_ be able to implement these new grid classes in the same way we have been without any `grid--#-col-tablet-down` type of class, if we wanted. And if we did, we wouldn't really need to refactor anything else to match in the future. Which would be nice because I'm also hesitant to introduce additional tech debt at this stage. 

I'm definitely interested in other devs view of this though, just in case I'm the only one who has concerns around this 🤷